products/podman-engine.md

```diff
@@ -0,0 +1,60 @@
+---
+title: Podman Engine
+category: app
+iconSlug: podman
+permalink: /podman-engine
+alternate_urls:
+-   /podman
+versionCommand: podman --remote version --format '{{.Server.Version}}'
+releaseDateColumn: true
+
+identifiers:
+-   repology: podman
+
+auto:
+  methods:
+  -   git: https://github.com/containers/podman.git
+      regex: ^v(?P<major>\d+)\.(?P<minor>\d+)\.(?P<patch>\d+)$
+
+releases:
+-   releaseCycle: "5.1"
+    releaseDate: 2024-05-29
+    eol: false
+    latest: "5.1.2"
+    latestReleaseDate: 2024-07-10
+
+-   releaseCycle: "5.0"
+    releaseDate: 2024-03-19
+    eol: true
+    latest: "5.0.3"
+    latestReleaseDate: 2024-05-10
+
+-   releaseCycle: "4.9"
+    releaseDate: 2024-01-22
+    eol: false
+    latest: "4.9.5"
+    latestReleaseDate: 2024-05-30
+
+-   releaseCycle: "4.8"
+    releaseDate: 2023-11-27
+    eol: true
+    latest: "4.8.3"
+    latestReleaseDate: 2024-01-03
+
+-   releaseCycle: "4.7"
+    releaseDate: 2023-09-27
+    eol: true
+    latest: "4.7.2"
+    latestReleaseDate: 2023-10-31
+
+---
+
+> [Podman](https://podman.io/) is a daemonless, open source, Linux native
+> tool designed to make it easy to find, run, build, share and deploy
+> applications using Open Container Initiative ([OCI](https://www.opencontainers.org/))
```

Comment:
```suggestion
> applications using [Open Container Initiative (OCI)](https://www.opencontainers.org/)
```

Reply:
Description was stolen from https://docs.podman.io/en/latest/, including the typo in the organization name